contributor/01-introduction-article.md

```diff
@@ -0,0 +1,97 @@
+# Draft comments
+
+ Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)
+* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
+* The [Lowering the Barriers to Entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.
+
+## Outline based Target
+### The Contributor in InnerSource
+
+  - Introduce the speaker. - only required for the video script I'd say. 
+  - Quick overview of the _Contributor_ role.
+  - Summary of the roles of InnerSource and how each relates to _Contributor_.
+  - Summary of the upcoming segments and what each covers.
+
+Outline for this section inspired by the [Trusted Committer introduction article.
+
+## Video Play-book based Target
+How do I best contribute to an InnerSource project?  This segment shares a summary of what it means to be an InnerSource Contributor.  We'll also give an overview of the upcoming training segments on being a great InnerSource Contributor.
+
+
+
+# Article
+
+## The Contributor in InnerSource.
+
+Without contributions, an InnerSource community and its project(s) would not exist in a meaningful way.
+There would be open repositories, ideally equipped with somewhat specific extra documents (e.g. a CONTRIBUTING.md and a README.md) and person(s) committing to them, e.g. a Trusted Committer.
+[EDITOR NOTE:
+- In other articles we draw a distinction between guest and host team.
+- We argue that the Committer is part of the guest team and needs to fit in with the host team.
+- In a more-than-one-person project what do we call the people aside of the TC/PO that are members of the host team? Just team members?
```

Comment:
I think that something like _host team members_ is fine.  Is this concept used somewhere in this article?

Reply:
We're using that concept implicitly in lines 41,42 and explicitly in line 55.
Also, we're announcing to use the concept extensively in line 85. 

And since home/guest might establish a mental concept of separation (us/them) I think that we should give that a aspect a consideration to see if and how we want to deal with it throughout the content. 
This could enhance clarity and prevent contradictory information later on. (Me being a nitpick and definition pedant here ;))

I've suggested the [Apache source below](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/98/files#r307215859) since that gives an example for this

Reply:
I think in a corporate context we assume that division is already built in by the organization.  We want to bring it together and suggest the terminology of _Host/Guest_ to encourage people to give warm welcome and collaboration to each other.

Reply:
Sounds good to me.
So we're using _host team member_ for a person that is part of the host team but no trusted committer (yet) then?

Reply:
Yes, I think so.